generators/client/templates/README.md.jhi.client.ejs

```diff
@@ -28,21 +28,20 @@
 _&>
 <%_
 let clientPackageMngrName = 'Npm';
+let clientPackageMngrWrapper = './npmw';
 let clientPackageMngrAddGlobal = 'install -g';
 let clientPackageMngrAdd = 'install --save --save-exact';
 let clientPackageMngrAddDev = 'install --save-dev --save-exact';
 -%>
-<&_ if (fragment.developmentSection) { -&>
```

Comment:
This is a multi-step template.
Is template control should be kept.

Reply:
Right.